patches/third_party-blink-renderer-core-loader-mixed_content_checker.h.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/third_party/blink/renderer/core/loader/mixed_content_checker.h b/third_party/blink/renderer/core/loader/mixed_content_checker.h
+index 402ce521e418425943d7bd88d57fc3d1d3832da5..a64e2ed54a0005245e46b10db08b3551569d6ab0 100644
+--- a/third_party/blink/renderer/core/loader/mixed_content_checker.h
++++ b/third_party/blink/renderer/core/loader/mixed_content_checker.h
+@@ -98,7 +98,6 @@ class CORE_EXPORT MixedContentChecker final {
+   static bool IsWebSocketAllowed(WorkerFetchContext&, const KURL&);
+ 
+   static bool IsMixedContent(const SecurityOrigin*, const KURL&);
+-  static bool IsMixedContent(const String& origin_protocol, const KURL&);
```

Comment:
this is intentional change to catch any use of the method during a rebase process.
we should handle it explicitly everywhere, otherwise mixed content may break in Tor again.